src/main/java/com/pyramidplundercounter/PyramidPlunderCounterPlugin.java

```diff
@@ -232,7 +255,9 @@ private void importData() {
                 chestLooted = importedData.getChestsLooted();
                 sarcoLooted = importedData.getSarcoLooted();
                 totalChance = importedData.getChanceOfBeingDry();
+				totalPetChance = importedData.getPetChanceOfBeingDry();
```

Comment:
wrong indent level

Reply:
My apologies, IntelliJ was set to 4-space tabs, fixed for my code in the latest commit.

Reply:
No worries king. I will submit the update to runelite so they can merge soon